app/src/main/res/layout/fragment_popup_attribute_list.xml

```diff
@@ -18,26 +18,68 @@
         xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
If we decide to rename `PopupAttributesListFragment ` to `PopupFragment`, we also might want to rename this to `fragment_popup.xml`

Reply:
updated